apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/BeneficiaryTransformerTest.java

```diff
@@ -270,6 +271,26 @@ public void transformSampleARecordWithoutLastUpdated() {
     assertMatches(beneficiary, patient, requestHeader);
   }
 
+  /**
+   * Verifies that {@link
+   * gov.cms.bfd.server.war.stu3.providers.BeneficiaryTransformer#transform(Beneficiary)} works as
+   * expected when run against the {@link StaticRifResource#SAMPLE_A_BENES} {@link Beneficiary} with
+   * a reference year field not found.
+   */
+  @Test
+  public void transformSampleARecordWithoutReferenceYear() {
```

Comment:
Similarly here, we should be checking for the newly gated MedicaidDualEligibility fields don't exist with no year.